test/csuite/schema_abort/main.c

```diff
@@ -541,18 +554,27 @@ thread_ckpt_run(void *arg)
      * Keep a separate file with the records we wrote for checking.
      */
     (void)unlink(ckpt_file);
-    memset(buf, 0, sizeof(buf));
+    memset(flush_tier_config, 0, sizeof(flush_tier_config));
+    memset(ts_config, 0, sizeof(ts_config));
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
     first_ckpt = true;
     ts = 0;
+    flush_tier = false;
+
+    testutil_check(__wt_snprintf(ts_config, sizeof(ts_config), "use_timestamp=true"));
+
+    testutil_check(__wt_snprintf(
+      flush_tier_config, sizeof(flush_tier_config), "flush_tier=(enabled,force),%s", ts_config));
```

Comment:
I find this slightly more confusing than the original code. I'd prefer `ckpt_config` and `ckpt_flush_config` as the string buffer names.

Also to be clear, this change isn't just a renaming, as this change also sets `force` all the time whereas the original code just used `enabled`.

Reply:
Fixed in f2a9620ccbcd7387228e9f631319df147093c77c